src/app/src/test/kotlin/org/vaccineimpact/api/tests/controllers/coverage/PopulatingCoverageTests.kt

```diff
@@ -14,6 +14,13 @@ import org.vaccineimpact.api.test_helpers.MontaguTests
 
 class PopulatingCoverageTests : MontaguTests()
 {
+    @Test
+    fun `can get coverage upload template`() {
+        val sut = CoverageController(mock(), mock(), PostDataHelper())
+        val result = sut.getCoverageUploadTemplate()
+        assertThat(result).isEqualTo("\"vaccine\", \"country\", \"activity_type\", \"gavi_support\", \"year\", \"age_first\", \"age_last\", \"gender\", \"target\", \"coverage\"")
```

Comment:
Wasn't there an extra flag for 'national' that was to go in? (In addition to the free text field which we agreed would go on the upload metadata.) One for a separate ticket?

Reply:
Yes, there's already a ticket for this in the sprint.